docs.html

```
@@ -0,0 +1,585 @@
+      </div>
+
+      <h3>Formato de Consulta</h3>
+      <p>El formato de búsqueda sigue este patrón:</p>
+
+      <pre><code>{ruta-partes} {separador} {búsqueda}
+        ↓          ↓          ↓
+    "raycast docs api - button"</code></pre>
+
+      <div class="example-box">
+        <strong>Ejemplo:</strong> Escribes <code>raycast docs api - hooks</code><br />
+        → Navega a <code>raycast > docs > api</code><br />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)